fe/fe-core/src/main/java/com/starrocks/sql/optimizer/OptimizerContext.java

```diff
@@ -43,6 +48,7 @@ public class OptimizerContext {
     private OptimizerTraceInfo traceInfo;
     private OptimizerConfig optimizerConfig;
     private List<MaterializationContext> candidateMvs;
+    private Map<List<Table>, List<MaterializedView>> usedMvs;
```

Comment:
why don't add in `MaterializationContext`?

Reply:
> why don't add it in `MaterializationContext`?

it is a context of query, not for mv.

Reply:
mv it into MaterializedContext